client/my-sites/backup/backup-storage-space/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+import { Card, ProgressBar } from '@automattic/components';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import {
+	BackupStorageSpaceUpsell,
+	BackupStorageSpaceUpsellOptions,
+} from './backup-storage-space-upsell';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const upsellLimit1 = 0.6;
+const upsellLimit2 = 0.85;
+
+const progressBarColors: Record< BackupStorageSpaceUpsellOptions, string > = {
+	no_upsell: '#2C3338',
+	first_upsell: '#DEB100',
+	second_upsell: '#E65054',
+	out_of_storage: '#E65054',
+};
```

Comment:
Good call. The `ProgressBar` has a `color` prop so I was using that, but it is better to keep the colors in CSS. The CSS is a bit weird due to the structure of `ProgressBar`, but it let me use the actual CSS variables instead of using JS strings.